File: Loginandshowdata/app/src/main/java/com/nitin/loginandshowdata/MainActivity.java
package com.nitin.loginandshowdata;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    EditText username, password;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        username = findViewById(R.id.user_name);
        password = findViewById(R.id.password);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
    }

    public void login(View view){
        String uname= username.getText().toString();
        String upass= password.getText().toString();

        Bundle bundle = new Bundle();

        bundle.putString("uname", uname);
        bundle.putString("upass", upass);

        Intent intent = new Intent(MainActivity.this,showUserDAta.class);

        intent.putExtras(bundle);
        startActivities(intent);

    }

}

File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val text = findViewById<TextView>(R.id.helloText)
        val button = findViewById<Button>(R.id.clickBtn)

        button.setOnClickListener {
            text.text = "Welcome to Android 🚀"
        }
    }
}
